ush/cam/evs_href_preppare.sh

```diff
@@ -18,7 +18,46 @@ nextday=`$NDATE +24 ${vday}09 |cut -c1-8`
 prevday=`$NDATE -24 ${vday}09 |cut -c1-8`
 
 #NOTE: COPYGB2 is using option -i3 (budget)   
+
+if [ $data = spcoutlook ] ; then
```

Comment:
Logan,

   SPCoutlook stats and plots are not included in href PR since Marcel has not put his output SPCoutlook mask files into
COMOUT directory. The script  evs_href_preppare.sh was copied to guthub but will not used in this PR. Current version of
evs_href_preppare.sh just uses Marcel's SPCoutlook mask files in hie personal account. When he put the mask files into
COMOUT directory, I'll update the path, or I will delete the script from prep directory?

Reply:
@BinbinZhou-NOAA this plan for updating the your script once masks are saved in COMOUT sounds good to me

Reply:
I will go ahead and resolve this conversation.